### PR TITLE
Extend the CentreonAPIContext to dump the result from the server

### PR DESCRIPTION
**Pull request by vhr**
Resolve BAM-438

### CODE REVIEW

File: src/behat/CentreonAPIContext.php [hunk 1]
```diff
@@ -206,8 +206,9 @@ public function responseHasKeys($property, PyStringNode $propVal)
     {
         $data = json_decode($this->getResponse()->getBody(true), true);
         if (!empty($data[$property])){
-            if ($data[$property] !== json_decode($propVal, true)){
-                throw new \Exception("Value of '".$property."' is not correct!\n");
+            if ($data[$property] !== json_decode($propVal, true)) {
+                throw new \Exception("Value of '".$property."' is not correct!\nExpecte to be: "
```
Comment (kduret) [inline src/behat/CentreonAPIContext.php]:
```suggestion
                throw new \Exception(
                    "Value of '" . $property . "' is not correct!\n"
                    . "Expected value: " . json_encode($data[$property]) . "\n"
```

Comment (ganoze) [inline src/behat/CentreonAPIContext.php]: **Expected**

File: src/behat/CentreonAPIContext.php [hunk 2]
```diff
@@ -206,8 +206,9 @@ public function responseHasKeys($property, PyStringNode $propVal)
...
+                    . json_encode($data[$property]) . "\n");
```
Comment (kduret) [inline src/behat/CentreonAPIContext.php]:
```suggestion
                );
```

File: src/behat/CentreonAPIContext.php [hunk 3]
```diff
@@ -205,9 +204,13 @@ public function responseHasProperty($property)
     public function responseHasKeys($property, PyStringNode $propVal)
     {
         $data = json_decode($this->getResponse()->getBody(true), true);
-        if (!empty($data[$property])){
-            if ($data[$property] !== json_decode($propVal, true)){
-                throw new \Exception("Value of '".$property."' is not correct!\n");
+        if (!empty($data[$property])) {
+            if ($data[$property] !== json_decode($propVal, true)) {
+                throw new \Exception(
+                    "Value of '{$property}' is not correct!\nExpecte to be: "
```
Comment (ganoze) [inline src/behat/CentreonAPIContext.php]:
```suggestion
                    "Value of '{$property}' is not correct!\nExpected to be: "
```

**Pull request merged**